nc-news/api.js

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://bc-nc-news-iv0p.onrender.com/api",
+});
+
+const getArticles = () => {
+  return api
+    .get("/articles")
+    .then(({ data }) => {
+      console.log(data.articles);
+      return data.articles;
+    })
+    .catch((error) => {
+      console.error("Error fetching articles:", error);
+      throw error; // Re-throw to handle it further up the call chain if necessary
+    });
```

Comment:
Nice work. You don't need a catch block though if you want to handle it elsewhere :)